open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/StyleFactory.java

```diff
@@ -39,10 +32,8 @@
     /** Logger reference. */
     private static final Logger LOGGER = Logger.getLogger(StyleFactory.class);
 
-    /**
-     * Applies the appropriate label styles on the created style.
-     */
-    private final LabelApplier myLabelApplier;
+    /* Applies the appropriate label styles on the created style. */
+//    private final LabelApplier myLabelApplier;

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.